Tests/xSharePoint/xSharePoint.xSPUserProfileServiceApp.Tests.ps1

```diff
@@ -12,7 +12,7 @@ $Global:CurrentSharePointStubModule = $SharePointCmdletModule
 $ModuleName = "MSFT_xSPUserProfileServiceApp"
 Import-Module (Join-Path $RepoRoot "Modules\xSharePoint\DSCResources\$ModuleName\$ModuleName.psm1")
 
-Describe "xSPUserProfileServiceApp" {
+Describe "MSFT_xSPUserProfileServiceApp" {
```

Comment:
Why did you change this? None of our other describe blocks contain the MSFT_ at the start